astroquery/cosmosim/core.py

```diff
@@ -641,7 +630,8 @@ def _starttime_dict(self):
                  for i in range(len(response_list))]
         self.starttime_dict = {}
         for i in range(len(soups)):
-            self.starttime_dict['{}'.format(completed_ids[i])] = '{}'.format(soups[i].find('uws:starttime').string)
+            self.starttime_dict['{}'.format(completed_ids[i])] = (
+                '{}'.format(soups[i].find('uws:starttime').string))
```

Comment:
Thanks for the heads up, there were quite many more of these. Doing str() is probably an overkill in many of the cases, but we need tests for the module before making more changes. I cannot make it work anyway, not even following the docs (that somehow fools the doctesting).